### DISCUSSION TITLE
Using Wear as a collector alongside Omnipod 5

### DISCUSSION THREAD

**Discussion by meatychunks99** [NightscoutFoundation/xDrip#3129] [Q&A]
Hi, having previously used a patched Huwawei Watch 2 as a collector very successfully, we're now seeing issues having moved to the Omnipod 5.
My understanding is that the phone collector uses the transmitter's bluetooth on slot 1, and the watch on slot 2. Slot 2 is now taken by the omnipod, so the watch can no longer connect.
I note that, under engineering mode, there is the ability to change the slot number for the phone. Is this possible to do for the watch? If so, how?
Thanks in advance.

**Comment by Navid200** [comment]
> Is this possible to do for the watch?  
  
I don't believe so.

Further, changing the slot number is behind engineering mode because it is not recommended, even from the phone.

**Reply by meatychunks99** [reply]
Could this please be considered as a feature?
The benefit to us would be that, using an OEM closed loop such as Omnipod 5 which takes the second bluetooth slot, means that my daughter must carry her phone in order to see her BG. She may as well look at her PDM/controller - which she rarely does. Having the BG on her watch means she's more likely to keep an eye on it - but as this stands she must always have her phone with her in order for the watch to work. Meaning she must carry the PDM and the phone now.
I imagine more users who use a wear device as a collector will encounter this issue as more people move to OEM looping pumps.

**Reply by Navid200** [reply]
Currently, there is no xDrip developer who is addressing WearOS issues or feature requests.  It will be wonderful if an interested party joins us who is willing to work on that.

Changing the slot number has resulted in damaging the transmitter.  One of the reasons something is behind engineering mode is that we cannot reasonably guarantee proper functionality.

I perfectly understand the need as you describe it.  
I can think of an alternative. 
BlueJay is a watch that might help in this case.  https://bluejay.website/

You can set BlueJay with the phone such that it would use the phone slot while the phone would use the the medical slot.  Then, if you disable collection on the phone, BlueJay should continue to collect from the phone slot.
Then, the pump and BlueJay should operate simultaneously.

Please note I don't have the pump and have not tested what I have described.

**Reply by Navid200** [reply]
In fact, you can use BlueJay GTS without a phone.  You can set it up from the watch to use the phone slot.

**Reply by meatychunks99** [reply]
Thanks @Navid200
